
# üëã Welcome to the Fried Lab!

#### We believe building **new** tools is the only way to break **old** barriers!

Science is unlikely to deliver transformative insights if we study systems and work in the same way we always have previously.  Therefore, answering challenging questions at the intersection of biophysics, cell biology, medicine, and sustainability requires new approaches and new models.  

We are improving science by ...  
- Developing methods that combine crosslinking, mass spectrometry, and NanoPore technology
- Using directed evolution to get proteins and nucleic acids to do our bidding
- Collaborating with other scientists from across the world
- Sharing data, maintaining science transparent and open, and placing inclusiveness at the core of our values

## üß¨ Our Science

### Refoldability of the Proteome
We have developed an approach using Limited-Proteolysis Mass Spectrometry to probe protein refolding for whole proteomes. Our studies suggest that many proteins‚Äô native structures may be kinetically trapped. In fact, we show that roughly half of the *E. coli* proteome cannot reassemble after chemical unfolding.  

[Learn More - ACS Paper](https://pubs.acs.org/doi/abs/10.1021/jacs.1c03270)  
[Learn More - BioRxiv](https://www.biorxiv.org/content/10.1101/2021.11.20.469408v1)

### Loop-able Translator
We have identified a method to circularize mRNA *in-vivo*, using permuted self-splicing introns, to direct translation of large repetitive protein sequences. In combination with engineered cellular secretion pathways, we are close to finalizing this technology in *B. subtilis*. This tool would enable the creation of sustainable, DNA-encodable, evolvable materials.  

[Learn More - ACS Paper](https://pubs.acs.org/doi/10.1021/acscentsci.1c00574)

### Co-translational protein folding *in-vivo*
We hypothesize that mRNA translation guides the protein folding process. Cells can coordinate synthesis of nascent proteins, folding, and assembly on the ribosome ‚Äì navigating a complicated free energy landscape. To probe the spatial-temporal complexities of nascent proteins, we use diazirene amino acids to capture nascent protein states *in-vivo* via Cross-Linking Mass Spectrometry. This approach provides helps us obtain structural information as proteins are synthesized on the ribosome.  

[Learn More - Scholar](https://scholar.google.com/citations?view_op=view_citation&hl=en&user=17Ov8cgAAAAJ&sortby=pubdate&citation_for_view=17Ov8cgAAAAJ:RtRctb2lSbAC)

### Structrual Bioinformatics 
We find that protein structure has evolved a lot slower than protein sequences. Finding pattern that are successful in nature is crucial to engineering new proteins for new technologies and therapeutics. This is why we have developed DomainMapper, which can take HMMER3 proteins alignments and return unique protein annotation for entire proteomes. With this tool we found that certain protein domains are more amenable to be split by nature and have other domains inserted within them.  

[Learn More - GitHub](https://github.com/FriedLabJHU/DomainMapper)
## üëÄ See what we are up to!
![https://twitter.com/](https://img.shields.io/twitter/follow/fried_lab?style=social)
[Lab Site](http://friedlab.com)
## üê¶ Johns Hopkins University
Remsen Hall Room 121  
3400 N Charles St  
Baltimore, MD, 21218

[Contact Us](mailto:sdfried@jhu.edu)


